src/header/searchport.rs

```diff
@@ -6,37 +6,36 @@ use hyper::header::{HeaderFormat, Header};
 const SEARCHPORT_HEADER_NAME: &'static str = "SEARCHPORT.UPNP.ORG";
 
 pub const SEARCHPORT_MIN_VALUE: u16 = 49152;
-pub const SEARCHPORT_MAX_VALUE: u16 = 65535;
 
 /// Represents a header used to specify a unicast port to send search requests to.
 ///
-/// If a SearchPort header is not included in a message then the device must
+/// If a `SearchPort` header is not included in a message then the device must
 /// respond to unicast search requests on the standard port of 1900.
 #[derive(Copy, Clone, PartialEq, Eq, Hash, Debug)]
 pub struct SearchPort(pub u16);
 
-unsafe impl Sync for SearchPort { }
+unsafe impl Sync for SearchPort {}
 
-unsafe impl Send for SearchPort { }
+unsafe impl Send for SearchPort {}
 
 impl Header for SearchPort {
     fn header_name() -> &'static str {
         SEARCHPORT_HEADER_NAME
     }
-    
+
     fn parse_header(raw: &[Vec<u8>]) -> error::Result<Self> {
         if raw.len() != 1 {
-            return Err(Error::Header)
+            return Err(Error::Header);
         }
-        
+
         let cow_str = String::from_utf8_lossy(&raw[0][..]);
-        
+
         let value = match u16::from_str_radix(&*cow_str, 10) {
             Ok(n) => n,
-            Err(_) => return Err(Error::Header)
+            Err(_) => return Err(Error::Header),
         };
-        
-        if value <= SEARCHPORT_MAX_VALUE && value >= SEARCHPORT_MIN_VALUE {
+
+        if value >= SEARCHPORT_MIN_VALUE {
```

Comment:
Ah yes, I have heard of ```clippy``` but never tried it out. I'll have to look into using it regularly.

Thanks!